website/routes.py

```diff
@@ -1243,11 +1243,13 @@ def make_url_map(app):
             '/project/<pid>/get_summary/',
             '/project/<pid>/node/<nid>/get_summary/',
         ], 'get', project_views.node.get_summary, json_renderer),
-
+        # Route "get_children" is deprecated. Use get_readable_descendants.
         Rule([
             '/project/<pid>/get_children/',
             '/project/<pid>/node/<nid>/get_children/',
```

Comment:
I highly doubt that any external parties are using this route, so it can probably be removed, as we're no longer using it. Not a blocker